README.md

```diff
@@ -3,6 +3,22 @@ NeoGradle
 
 Minecraft mod development framework used by NeoForge and FML for the Gradle build system.
 
-For a quick start, see how the [NeoForge Mod Development Kit](https://github.com/neoforged/MDK) uses NeoGradle, or see our official [Documentation](https://docs.neoforged.net/neogradle/docs/).
+For a quick start, see how the [NeoForge Mod Development Kit](https://github.com/neoforged/MDK) uses NeoGradle, or see
+our official [Documentation](https://docs.neoforged.net/neogradle/docs/).
+
+To see the latest available version of NeoGradle, visit
+the [NeoForged Maven](https://maven.neoforged.net/#/releases/net/neoforged/gradle/userdev).
```

Comment:
This should link to `https://projects.neoforged.net/neoforged/neogradle` as it provides copy buttons.